lib/viking/view.js

```diff
@@ -138,7 +138,13 @@ export default class View extends EventBus {
         if (this.el) {
             this.setElement(result(this, 'el'));
         } else {
-            const attrs = Object.assign({}, result(this, 'attributes')); //: {id?: string, class?: string}
+            let attrs;
+            // check if attributes is designated assignableProperties, thus intended for another purpose
+            if (!scanPrototypesFor(this.constructor, 'assignableProperties').flat().includes('attributes')) {
+                attrs = Object.assign({}, result(this, 'attributes')); //: {id?: string, class?: string}
+            } else {
+                attrs = {}
+            }
```

Comment:
```suggestion
            let attrs = {};
            
            // if attributes is not in assignableProperties it is assumed the user is using it
            if (!scanPrototypesFor(this.constructor, 'assignableProperties').flat().includes('attributes')) {
                Object.assign(attrs, result(this, 'attributes'));
            }
```

Reply:
Actually just skip the `each` where it assigns them to the element too because if it's in `assignableProperties` then it doesn't need to do the loop